### PR TITLE
RaspyRFM platform documentation

### PR DESCRIPTION
**Pull request by markusressel**
**Description:**
Added support for 433MHz devices controlled via ConnAir/ITGW-433/RaspyRFM gateway.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#19130

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/switch.raspyrfm.markdown [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+sharing: true
+footer: true
+logo: seegelsysteme.png
+ha_category: Switch
+ha_release: 0.85
+---
+
+The `raspyrfm` component adds support for cheap RC 433 MHz outlets via one of the supported gateways.
+
+Initially this component was created to support the Simple Solutions `ConnAir` gateway which has been discontinued. There are custom alternatives that reimplemented the protocol used by the ConnAir though like this [ConnAir emulator](https://github.com/Phunkafizer/RaspyRFM/blob/master/connair.py) wich can be used in conjunction with the [RaspyRFM-II](https://www.seegel-systeme.de/produkt/raspyrfm-ii) rc module for a Raspberry Pi.
```
Comment (MartinHjelmare) [inline source/_components/switch.raspyrfm.markdown]: which can be used

File: source/_components/switch.raspyrfm.markdown [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+Other vendors of 433 MHz rc outlets have also created gateways that use a very similar protocol and can also be used with this component like the Intertechno [ITGW-433 LAN Gateway](https://www.intertechno24.de/LAN-Gateway/Gateway-ITGW-433.html) 
+ 
+
+```yaml
+# Example configuration.yaml entry
+switch:
+  platform: raspyrfm
+  gateway_manufacturer: Seegel Systeme
+  gateway_model: RaspyRFM
+  host: 127.0.0.1              # Optional
```
Comment (MartinHjelmare) [inline source/_components/switch.raspyrfm.markdown]: The first config example should be minimal and only include required items.

File: source/_components/switch.raspyrfm.markdown [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+  default: 127.0.0.1
+  type: string
+host:
+  description: Port of the gateway.
+  required: false
+  default: depends on the gateway model
+  type: integer
+switches:
+  description: List of switches that can be controlled with this gateway.
+  required: false
```
Comment (MartinHjelmare) [inline source/_components/switch.raspyrfm.markdown]: This is required.

File: source/_components/switch.raspyrfm.markdown [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+  default: []
```
Comment (MartinHjelmare) [inline source/_components/switch.raspyrfm.markdown]: Remove default.

File: source/_components/switch.raspyrfm.markdown (hunk 1 above)
Comment (markusressel) [reply source/_components/switch.raspyrfm.markdown]: fixed

File: source/_components/switch.raspyrfm.markdown (hunk 3 above)
Comment (markusressel) [reply source/_components/switch.raspyrfm.markdown]: Ok

File: source/_components/switch.raspyrfm.markdown (hunk 4 above)
Comment (markusressel) [reply source/_components/switch.raspyrfm.markdown]: Ok

File: source/_components/switch.raspyrfm.markdown (hunk 2 above)
Comment (markusressel) [reply source/_components/switch.raspyrfm.markdown]: Ok

File: source/_components/switch.raspyrfm.markdown [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+sharing: true
+footer: true
+logo: seegelsysteme.png
+ha_category: Switch
+ha_release: 0.85
+---
+
+The `raspyrfm` component adds support for cheap RC 433 MHz outlets via one of the supported gateways.
+
+Initially this component was created to support the Simple Solutions `ConnAir` gateway which has been discontinued. There are custom alternatives that reimplemented the protocol used by the ConnAir though like this [ConnAir emulator](https://github.com/Phunkafizer/RaspyRFM/blob/master/connair.py) wich can be used in conjunction with the [RaspyRFM-II](https://www.seegel-systeme.de/produkt/raspyrfm-ii) rc module for a Raspberry Pi which can be used.
```
Comment (MartinHjelmare) [inline source/_components/switch.raspyrfm.markdown]: Nah, I was referring to the existing phrase that had a typo in "which".

Comment (markusressel) [reply source/_components/switch.raspyrfm.markdown]: :man_facepalming: fixed it :laughing:

**Review by MartinHjelmare — APPROVED** [review]
Great!

File: source/_components/switch.raspyrfm.markdown [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+description: "Instructions on how to integrate RaspyRFM switches into Home Assistant."
+date: 2018-12-08
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: seegelsysteme.png
+ha_category: Switch
+ha_release: 0.85
+---
```
Comment (MartinHjelmare) [inline source/_components/switch.raspyrfm.markdown:14]: Add `ha_iot_class`.

**Review by MartinHjelmare — DISMISSED** [review]
Sorry, I missed one thing.

File: source/_components/switch.raspyrfm.markdown (hunk 6 above)
Comment (markusressel) [reply source/_components/switch.raspyrfm.markdown:14]: fixed

File: source/_components/switch.raspyrfm.markdown [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+description: "Instructions on how to integrate RaspyRFM switches into Home Assistant."
+date: 2018-12-08
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: seegelsysteme.png
+ha_category: Switch
+ha_release: 0.85
+ha_iot_class: "Local Polling"
```
Comment (MartinHjelmare) [inline source/_components/switch.raspyrfm.markdown]: Assumed State

Comment (markusressel) [reply source/_components/switch.raspyrfm.markdown]: fixed

File: source/_components/switch.raspyrfm.markdown [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+Initially this component was created to support the Simple Solutions `ConnAir` gateway which has been discontinued. There are custom alternatives that reimplemented the protocol used by the ConnAir though like this [ConnAir emulator](https://github.com/Phunkafizer/RaspyRFM/blob/master/connair.py) which can be used in conjunction with the [RaspyRFM-II](https://www.seegel-systeme.de/produkt/raspyrfm-ii) rc module for a Raspberry Pi.
+
+Other vendors of 433 MHz rc outlets have also created gateways that use a very similar protocol and can also be used with this component like the Intertechno [ITGW-433 LAN Gateway](https://www.intertechno24.de/LAN-Gateway/Gateway-ITGW-433.html) 
+ 
+
+```yaml
+# Example configuration.yaml entry
+switch:
+  platform: raspyrfm
+  gateway_manufacturer: Seegel Systeme
```
Comment (frenck) [inline source/_components/switch.raspyrfm.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/switch.raspyrfm.markdown [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+  gateway_model: RaspyRFM
```
Comment (frenck) [inline source/_components/switch.raspyrfm.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_components/switch.raspyrfm.markdown (hunk 8 above)
Comment (markusressel) [reply source/_components/switch.raspyrfm.markdown]: removed

File: source/_components/switch.raspyrfm.markdown (hunk 9 above)
Comment (markusressel) [reply source/_components/switch.raspyrfm.markdown]: removed

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @markusressel 👍

**Pull request merged**